src/lib/components/ContentfulRichText/nodes/EntryHyperlink.svelte

```diff
@@ -0,0 +1,39 @@
+<script lang="ts">
+  import type { Node as NodeType, EntryHyperlink } from "@contentful/rich-text-types";
+  import type { PageMetadataMapItem } from "$lib/loadPageMetadataMap";
+  import Node from "./Node.svelte";
+  import { getContext } from "svelte";
+  import { linksKey, type LinksContext } from "../context";
+  import { isEntryHyperlink } from "../predicates";
+  import ContactCard from "$lib/components/ContactCard";
+
+  export let node: NodeType;
+  if (!isEntryHyperlink(node)) {
+    throw new Error("node is not an entry hyperlink");
+  }
+  let entryHyperlink: EntryHyperlink = node;
+
+  const linksContext = getContext<LinksContext | undefined>(linksKey);
+  if (!linksContext) throw new Error("no context was provided for entry hyperlink");
+  const { id: entryID } = entryHyperlink.data.target.sys;
+
+  const entry = linksContext.linksEntriesMaps.hyperlink.get(entryID);
+
+  let entryMetadata: PageMetadataMapItem;
+  if (entry?.__typename === "PageMetadata") {
+    const entryMetadata = linksContext.pageMetadataMap.get(entryID);
+    if (!entryMetadata) {
+      throw new Error(`the entry asset ${entryID} was not found in the pageMetadataMap`);
+    }
+  }
+</script>
+
+{#if entry?.__typename === "PageMetadata"}
+  <a href={entryMetadata?.url}
```

Comment:
nit: Use `Link` component. Will help us out in the future (we'll probably want to determine whether `rel="external"` programmatically to start indiicating when a user is leaving the site).